src/cdk-experimental/scrolling/auto-size-virtual-scroll.ts

```diff
@@ -464,6 +464,6 @@ export class CdkAutoSizeVirtualScroll implements OnChanges {
     this._scrollStrategy.updateBufferSize(this.minBufferPx, this.maxBufferPx);
   }
 
-  static ngAcceptInputType_minBufferPx: number | string;
-  static ngAcceptInputType_maxBufferPx: number | string;
+  static ngAcceptInputType_minBufferPx: number | string | null | undefined;
```

Comment:
Maybe we should make this `static ngAcceptInputType_minBufferPx?: number | string | null;` so it's a bit shorter?

Reply:
Thought about this too, and also tested if that works with `typeof X.ngAcceptInputType_disabled`, but I wanted to make it explicit since we need to specify `null` anyway and the optional member syntax is less obvious IMO.

Reply:
Happy to change if you feel strong about it. I don't see anything wrong with being explicit here. The size won't affect runtime anyway.